<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorMarker</class>
 <widget class="QWidget" name="InspectorMarker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="accessibleName">
   <string>Marker Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>17</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Marker</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="2">
      <widget class="QToolButton" name="resetMarkerType">
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Marker type' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="jumpLabel">
       <property name="accessibleName">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="resetJumpLabel">
       <property name="toolTip">
        <string>Reset to default</string>
       </property>
       <property name="accessibleName">
        <string>Reset 'Label' value</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../musescore.qrc">
         <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="markerType">
       <property name="accessibleName">
        <string>Marker Type</string>
       </property>
       <item>
        <property name="text">
         <string>Segno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Segno variation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Varied coda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Codetta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>To Coda</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Custom</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Marker type</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>markerType</tabstop>
  <tabstop>resetMarkerType</tabstop>
  <tabstop>jumpLabel</tabstop>
  <tabstop>resetJumpLabel</tabstop>
 </tabstops>
 <resources>
  <include location="../musescore.qrc"/>
 </resources>
 <connections/>
</ui>
